.github/workflows/want-lgtm-all.yml

```diff
@@ -0,0 +1,165 @@
+name: 'want-lgtm-all'
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+      - ready_for_review
+      - review_requested
+      - review_request_removed
+  pull_request_review:
+    types:
+      - submitted
+      - dismissed
+      - edited
+  workflow_call:
+  workflow_dispatch:
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+permissions:
+  actions: 'write'
+  pull-requests: 'read'
+
+jobs:
+  validate:
+    runs-on: 'ubuntu-latest'
+    steps:
+      - id: 'validate-event'
+        name: 'Validate GitHub Event'
+        uses: 'actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410' # ratchet:actions/github-script@v6
+        with:
+          script: |-
+            // validate required event names
+            if (!["pull_request", "pull_request_review"].includes(context.eventName)) {
```

Comment:
Can we do this as an `if` on the job instead?

Reply:
Sorry, I'm still not seeing why we can't use a job `if`?

Reply:
I think you need to refresh.....

Reply:
You are correct.